core/src/Kokkos_CopyViews.hpp

```diff
@@ -1298,9 +1298,11 @@ inline std::enable_if_t<
 contiguous_fill_or_memset(
     const ExecutionSpace& exec_space, const View<DT, DP...>& dst,
     typename ViewTraits<DT, DP...>::const_value_type& value) {
+#ifndef KOKKOS_ARCH_A64FX
```

Comment:
Do you really need to disable it here, too? This is also used for `Kokkos::deep_copy(view, 0)`. Changing `core/src/impl/Kokkos_ViewMapping.hpp` should be enough to avoid calling memset in the View initialization.

Reply:
it will screw up withoutviewinit followed by deep_copy(a,0) later

Reply:
Is that the usual case? If you specify `Kokkos::WithoutInitializing` you wouldn't normally run `deep_copy(a,0)` later but fill the View in another way, right?
My thinking is that you would normally call `deep_copy(a,0)` for a VIew that has already been used, or am I wrong?

Reply:
I know of a number of apps where that doesn't happen. Basically you know that your normal algorithm will anyway reset the data because you call the thing multiple times (like timesteps or other types of iterations). So you tell yourself: I gonna reset this thing anyway, so during allocation (or more often than not resizing as part of load balancing etc.) you avoid initialization since you know you will overwrite.

Reply:
If you think we should do it here as well, that's fine with me.